migrations/versions/2f75e96dd995_.py

```diff
@@ -0,0 +1,59 @@
+"""empty message
+
+Revision ID: 2f75e96dd995
+Revises: 4bc7f3b6b092
+Create Date: 2019-08-28 17:33:02.279980
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '2f75e96dd995'
+down_revision = '4bc7f3b6b092'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute('ALTER TABLE IF EXISTS upload_jobs RENAME TO assay_uploads')
+    op.execute('ALTER TABLE IF EXISTS assay_uploads RENAME COLUMN metadata_json_patch TO assay_patch')
+    op.execute('ALTER SEQUENCE IF EXISTS upload_jobs_id_seq RENAME TO assay_uploads_id_seq')
+    op.execute('ALTER INDEX IF EXISTS upload_jobs_pkey RENAME TO assay_uploads_pkey')
+    op.execute('ALTER INDEX IF EXISTS gcs_objects_idx RENAME TO ix_assay_uploads_gcs_file_uris')
+    op.add_column('assay_uploads', sa.Column('trial_id', sa.String(), nullable=False))
```

Comment:
I think you don't want to create non nullable column with out default value.
I would do it in three steps
```suggestion
    op.add_column('assay_uploads', sa.Column('trial_id', sa.String(), nullable=True))
    op.execute("UPDATE assay_uploads SET trial_id = metadata_json->>'lead_organisation_study_di'")
    op.alter_column('assay_uploads', 'trial_id',
               existing_type=sa.VARCHAR(),
               nullable=False)
```